ui/app/AppLayouts/Wallet/views/LeftTabView.qml

```diff
@@ -51,6 +52,54 @@ Rectangle {
         }
     }
 
+    Loader {
+        id: walletBckgAccountContextMenu
+        sourceComponent: AccountContextMenu {
+            onAddNewAccountClicked: {
+                RootStore.runAddAccountPopup()
+            }
+
+            onAddWatchOnlyAccountClicked: {
+                RootStore.runAddWatchOnlyAccountPopup()
+            }
+        }
+    }
+
+    Loader {
+        id: removeAccountConfirmation
+        active: false
+
+        property bool simple
+        property string accountKeyUid
+        property string accountName
+        property string accountAddress
+        property string accountDerivationPath
+
+        sourceComponent: RemoveAccountConfirmationPopup {
+            anchors.centerIn: parent
+            width: 400
+
+            simple: removeAccountConfirmation.simple
+            accountKeyUid: removeAccountConfirmation.accountKeyUid
+            accountName: removeAccountConfirmation.accountName
+            accountAddress: removeAccountConfirmation.accountAddress
+            accountDerivationPath: removeAccountConfirmation.accountDerivationPath
+
+            onClosed: {
+                removeAccountConfirmation.active = false
+            }
+
+            onRemoveAccount: {
+                close()
+                RootStore.deleteAccount(keyUid, address)
```

Comment:
@IvanBelyakoff theoretically it may fail (but the only reason to fail is if something gets wrong with the DB, in that case I am not sure that we can do anything to recover the db/app), but even if it fails we should display a new popup informing user that the action failed, not keep this one opened.